### PR TITLE
Change for customer Name

### PR DESCRIPTION
**Pull request by talktovikas**
### Description
This is for https://progresssoftware.atlassian.net/browse/CHEF-16635

[Please describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

**Comment by talktovikas** [comment]
Adhoc Pipeline for this: https://buildkite.com/chef/chef-chef-server-main-omnibus-adhoc/builds/6790

File: src/oc_erchef/apps/chef_telemetry/src/chef_telemetry_worker.erl [hunk 1]
```diff
@@ -199,29 +199,51 @@ solr_search(Query) ->
             {Error, Reason}
     end.
 
+get_license_company_name()->
+    {_Lic, _Type, _GracePeriod, _ExpDate, _Msg, CN,_LID}  = gen_server:call(chef_license_worker, get_license),
+    CN.
+
+determine_license_id()->
+    {_Lic, _Type, _GracePeriod, _ExpDate, _Msg, _CN, LicenseID}  = gen_server:call(chef_license_worker, get_license),
```
Comment (jashaik) [inline src/oc_erchef/apps/chef_telemetry/src/chef_telemetry_worker.erl:207]: chef_license:get_license() 
```suggestion
    {_Lic, _Type, _GracePeriod, _ExpDate, _Msg, _CN, LicenseID}  = chef_license:get_license(),
```

File: src/oc_erchef/apps/chef_telemetry/test/chef_telemetry_worker_test.erl [hunk 2]
```diff
@@ -28,15 +28,21 @@
...
+field_value_test() ->
     State = #state{fqdn_select = {ok, [[{<<"property">>, <<"FQDN:node1.domain1.com">>}],
                                        [{<<"property">>, <<"FQDN:node2.subdomain2.domain2.com">>}],
                                        [{<<"property">>, <<"FQDN:node3.subdomain3.domain3.co.uk">>}]]},
         should_send = true,
         user_emails = [[{<<"email">>, <<"test@testorg.com">>}]],
         nodes_count = 10
     },
-    Expected = #expected{company_name = <<"testorg">>,
+    {_Lic, _Type, _GracePeriod, _ExpDate, _Msg, CN,_}  = gen_server:call(chef_license_worker, get_license),
```
Comment (jashaik) [inline src/oc_erchef/apps/chef_telemetry/test/chef_telemetry_worker_test.erl:39]:
```suggestion
    {_Lic, _Type, _GracePeriod, _ExpDate, _Msg, CN,_}  = chef_license:get_license(),
```

File: src/oc_erchef/apps/chef_telemetry/test/chef_telemetry_worker_test.erl [hunk 3]
```diff
@@ -135,12 +141,25 @@ get_message() ->
        throw(no_request)
     end.
 
+determine_license_id()->
+    {_Lic, _Type, _GracePeriod, _ExpDate, _Msg, _CN, LicenseID}  = gen_server:call(chef_license_worker, get_license),
```
Comment (jashaik) [inline src/oc_erchef/apps/chef_telemetry/test/chef_telemetry_worker_test.erl:145]:
```suggestion
    {_Lic, _Type, _GracePeriod, _ExpDate, _Msg, _CN, LicenseID}  = chef_license:get_license(),
```

**Review by jashaik — APPROVED** [review]

**Pull request merged**